app/scripts/controllers/authentication/authentication-controller.ts

```diff
@@ -105,6 +116,25 @@ export default class AuthenticationController extends BaseController<
 > {
   #metametrics: MetaMetricsAuth;
 
+  #isUnlocked = false;
+
+  #keyringController = {
+    setupLockedState: () => {
```

Comment:
Same comment as in core, maybe for readability:

```suggestion
    setupLockedStateSubscriptions: () => {
```

Reply:
As we need to add tests for test coverage, I'll update to this.